bin/cml/runner.js

```diff
@@ -559,9 +568,9 @@ exports.builder = (yargs) =>
       },
       destroyDelay: {
         type: 'number',
-        default: 20,
+        default: 10,
         hidden: true,
-        description: 'Destroy delay'
+        description: 'Seconds to wait for collecting logs on failure (https://github.com/iterative/cml/issues/413)'
```

Comment:
Why do not we wait for me to fix it? I have never worked this way before.

Reply:
Whoops; sorry! Feel free to revert the commit.

Reply:
176ccf2 :)